apollo-router-core/src/query_planner/mod.rs

```diff
@@ -69,6 +74,8 @@ impl PlanNode {
             Self::Sequence { nodes } => nodes.iter().any(|n| n.contains_mutations()),
             Self::Parallel { nodes } => nodes.iter().any(|n| n.contains_mutations()),
             Self::Fetch(fetch_node) => fetch_node.operation_kind() == &OperationKind::Mutation,
+            // FIXME: not sure about that one, can there be mutations in the deferred part?
```

Comment:
Regarding the official RFC https://github.com/graphql/graphql-wg/blob/main/rfcs/DeferStream.md there is nothing about mutation

Reply:
deferred mutations sound like a potential footgun tbh